hackathon_site/dashboard/frontend/src/components/dashboard/TeamCard/TeamCard.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import styles from "./TeamCard.module.scss";
+import Paper from "@material-ui/core/Paper";
+import Link from "@material-ui/core/Link";
+import Container from "@material-ui/core/Container";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+
+const TeamCard = ({ members, teamCode, EditButton }) => {
+    const title = teamCode === "" ? "Team" : "Team " + teamCode;
+
+    return (
+        <Container className={styles.TeamCard} maxWidth={false} disableGutters={true}>
```

Comment:
Change this to: `<Grid className={styles.TeamCard} item>`. The TeamCard is wrapped in a `<Grid>` component within the dashboard. You'll have to import Grid.